tkg/client/cluster.go

```diff
@@ -198,6 +198,7 @@ func (c *TkgClient) CreateCluster(options *CreateClusterOptions, waitForCluster
 	if !waitForCluster {
 		return false, nil
 	}
+	log.Infof("... Done requesting creation of workload cluster '%s'.  Will now wait for it's completion...")
```

Comment:
missing a parameter for the `%s`, should be `options.ClusterName` as well?

Reply:
fixed